DiamondLightSource/SynchWeb
# [LIMS-80]Feature: Tidy responsive scheduling questions

**JIRA ticket:** [LIMS-80](https://jira.diamond.ac.uk/browse/LIMS-80)

**Changes:**
- Remove use of shipments_comments.json - remove questions from within the comments field, replacing them with separate fields
- Add extra fields to the create shipment form, for all dynamically-scheduled remote visits: Session length, Energy/wavelength, Microfocus beam, Scheduling restrictions, Last minute beam time, Dewar grouping
- Add an extra field for industrial dynamically-scheduled remote visits: Remote or mail in?
- Add extra fields for all shipments: Tools enclosed?, Hard drive enclosed?
- Add extra rows to the shipment page corresponding to the new fields

**To test:**
- Check that the correct fields appear depending on the circumstances (dynamic scheduling, industrial visit, etc.)
- Check that the input for these fields appears correctly on the shipment page once submitted

**Note:** The input of the new fields is stored as JSON in the 'extra' column of the Shipping table in ISPyB
